packages/calcite-components/src/components/color-picker-hex-input/color-picker-hex-input.e2e.ts

```diff
@@ -272,11 +272,10 @@ describe("calcite-color-picker-hex-input", () => {
     await selectText(input);
     const longhandHexWithExtraChars = "bbbbbbbbc";
     await page.keyboard.type(longhandHexWithExtraChars);
-    await page.keyboard.press("Enter");
```

Comment:
question: This test is not testing the Enter key any longer. Is that intentional?

Reply:
I removed the Enter key since values are now auto-committed when typing/pasting a valid hex code. Do you think I should add it back?

Reply:
No, that sounds reasonable. 👍